circuits/ts/__tests__/Hasher.test.ts

```diff
@@ -191,7 +191,7 @@ describe('Poseidon hash circuits', () => {
         const circuit = 'messageHasher_test'
         it('correctly hashes a message', async () => {
             const k = new Keypair()
-            const random50bitBigInt = (): BigInt => {
+            const random50bitBigInt = (): bigint => {
```

Comment:
For future reference, I'd argue this `BigInt` --> `bigint` refactor should be a single PR. Would make this more manageable to review.

Reply:
Sure, I can do the rest of the repo in one PR, would be great to have this merged first though